browser/scripts/ui-site.js

```diff
@@ -84,7 +87,7 @@ var siteUI = new function() {
 		VizorUI.enablePopupEmbedLinks($body);
 
 		var signupCallback = function() {	// slush
-			window.location.href = '/edit';
```

Comment:
do we really want to change this to '/account' instead of going straight to the editor?